.github/workflows/main.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+    branches-ignore:
+      - "releases/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        plone-version:
+          - '5.0'
+          - '5.1'
+          - '5.2'
+        python-version: ['2.7', '3.8']
+        exclude:
+          # Only 5.2+ runs on py3
+          - plone-version: 5.0
+            python-version: 3.6
+          - plone-version: 5.0
+            python-version: 3.7
+          - plone-version: 5.0
+            python-version: 3.8
+          - plone-version: 5.1
+            python-version: 3.6
+          - plone-version: 5.1
+            python-version: 3.7
+          - plone-version: 5.1
+            python-version: 3.8
```

Comment:
I think you could add the 5.2 + python 3.8 combo using `include` instead of excluding the others: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-new-combinations